styles/chess.com/catppuccin.user.css

```diff
@@ -0,0 +1,595 @@
+      --globalBackgroundOpaque: @mantle !important;
+      --globalSecondaryBackground: @crust !important;
+      --globalTertiaryBackground: @surface0 !important;
+      --globalAccentBackground: @accent-colour !important;
+      --globalSecondaryAccentBackground: darken(@accent-colour, 20%) !important;
+      --globalSiteBackground: @base !important;
+      --subtleButtonBackground: hsla(0, 0%, 100%, 0.16);
+      --globalBorder: @surface0 !important;
+      --globalGray: hsla(0, 0%, 100%, 0.08);
+      --globalGraySoft: hsla(0, 0%, 100%, 0.08);
+      --globalColorThemeFull: #fff;
+      --globalColorThemeHigh: hsla(0, 0%, 100%, 0.85);
+      --globalColorThemeMid: @overlay2 !important;
+      --globalColorThemeLow: @overlay0 !important;
+      --globalColorThemeLower: @base !important;
+      --globalColorThemeLink: @sapphire !important;
+      --globalColorThemeBlueToWhite: #fff;
+      --globalColorThemeBlueToMid: hsla(0, 0%, 100%, 0.72);
+      --globalColorThemeBlueToMidHover: hsla(0, 0%, 100%, 0.85);
+      --globalColorThemeBlueToHigh: hsla(0, 0%, 100%, 0.85);
+      --globalColorThemeBlueToHighHover: #fff;
+      --globalColorThemeHighToMid: hsla(0, 0%, 100%, 0.72);
+      --globalColorThemeFullToMid: hsla(0, 0%, 100%, 0.72);
+      --globalColorNeutral50: rgba(0, 0, 0, 0.1);
+      --globalColorNeutral100: rgba(0, 0, 0, 0.2);
+      --globalColorNeutral200: rgba(0, 0, 0, 0.4);
+      --globalOverlayBackground: @base !important;
+      --globalColorWin: @green !important;
+      --globalColorDraw: @accent-colour !important;
+      --globalColorLoss: @red !important;
+    }
+
+    /* index page */
+    .promo-title,
+    .index-title {
+      color: @text !important;
+    }
+
+    .index-info-item-counter {
+      color: @text !important;
+    }
+
+    .index-info-item {
+      color: @subtext0 !important;
+    }
+
+    .index-quote-author .index-chess-title {
+      background: @accent-colour !important;
+      color: @crust !important;
+    }
+
+    .authentication-intro-levels {
+      --labelBackground: @surface0 !important;
+      --labelBackgroundHover: @surface1 !important;
+
+      .authentication-intro-level-v5.authentication-intro-selected {
+        border-color: @accent-colour !important;
+      }
+    }
+
+    /* ui */
+    .user-username-component {
+      color: @text;
+    }
+
+    .user-tagline-rating.user-tagline-white {
+      color: @overlay1;
+    }
+
+    .nav-link-text {
+      color: @text !important;
+    }
+    .nav-menu-area:last-of-type .icon-font-chess {
+      color: @mantle !important;
+    }
+    #sb .nav-popover.dark .btn-link {
+      color: @text;
+    }
+    #sb .nav-menu-area:last-of-type .icon-font-chess {
+      color: @text !important;
+    }
+    .nav-link-main-design:hover,
+    .nav-link-main-link:hover {
+      color: darken(@text, 20%) !important;
+    }
+    .nav-link-main-design,
+    .nav-link-main-link {
+      color: @text !important;
+    }
+    .ui_v5-button-component {
```

Comment:
the folowing or something similar should be added here
```diff
+ --boxShadow: none;
+ --boxShadowHover: none;
```
to prevent this from occuring:
![image](https://github.com/catppuccin/userstyles/assets/71222764/8affa0a0-9258-4c0c-ba93-64a13b912064)